automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala

```diff
@@ -51,4 +52,50 @@ class DataLibrarySpec extends FreeSpec with WebBrowserSpec with UserFixtures wit
       }
     }
   }
+
+  "For a dataset with facets" in withWebDriver { implicit driver =>
+    val curatorUser = UserPool.chooseCurator
+    implicit val authToken: AuthToken = curatorUser.makeAuthToken()
+    withWorkspace(namespace, "Facets", attributes = Some(WorkspaceData.tags)) { wsName =>
+      withCleanUp {
+        //Hard Coded titles in our UI code
+        val cohortPhenotypeIndicationSection = "Cohort Phenotype/Indication"
+        val experimentalStrategySection = "Experimental Strategy"
+        val projectNameSection = "Project Name"
+        val primaryDiseaseSiteSection = "Primary Disease Site"
+        val dataUseLimitationSection = "Data Use Limitation"
+
```

Comment:
If you think future tests will want to reference these same strings, maybe move them to a common area. Maybe in the Library Page class we can have a facets or search class...